ambari-server/src/test/java/org/apache/ambari/server/controller/internal/ViewInstanceResourceProviderTest.java

```diff
@@ -71,6 +71,7 @@
 
   @Before
   public void before() {
+    ViewRegistry.initInstance(viewregistry);
```

Comment:
Do we still need the `static` init?

Reply:
You are correct, the static block is no longer necessary. I have therefore removed it from all three classes, and all tests in these classes still pass.